examples/resources/anxcloud_lbaas_loadbalancer/resource.tf

```diff
@@ -0,0 +1,4 @@
+resource "anxcloud_lbaas_loadbalancer" "example" {
+  name       = "example-lb"
+  ip_address = "192.0.2.1"
```

Comment:
we should maybe add some docs how to create the LoadBalancer - at least noting that "Just creating a LoadBalancer resource will sadly not give you a functional LoadBalancer, only make an existing one configurable via Engine"